website/docs/cloud-docs/users-teams-organizations/single-sign-on/index.mdx

```diff
@@ -46,16 +46,17 @@ If an organization's owners disable SSO (or downgrade the organization's account
 
 1. Provide your organization name and click "Next".
 
+**SCREENSHOT FOR DELETION**
    ![Screenshot: The Terraform Cloud SSO sign-in page, asking for an organization name.](/img/docs/sign-in.png)
 
 1. If you've signed in to Terraform Cloud with SSO before, proceed to the next step.
 
    If you're signing in for the first time under this account or for the first time accessing this organization, you'll be required to create a new account or link to an existing account. Use the links below the account creation form if you want to link your SSO identity to an existing account, then fill out and submit the relevant form.
-
+**SCREENSHOT FOR DELETION**
    ![Screenshot: Terraform Cloud's create account page.](/img/docs/create-account.png)
-
+**SCREENSHOT FOR DELETION**
```

Comment:
so something really weird is that I don't see these in the actual docs right now: https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/single-sign-on

hmmm is your branch up-to-date with main?